src/Components/PostsComponent/DisplayPosts.jsx

```diff
@@ -1,10 +1,52 @@
 import React from "react"
 import getAllPosts from "../../helpers/allPosts"
+import createNewPost from "../../helpers/CreatePosts"
+import getUniquePost from "../../helpers/getUniquePost"
+import CreatePost from "./CreatePost"
+import sortPosts from "../../helpers/SortPosts"
+// import SortPosts from "./SortPosts"
 
 class DisplayPosts extends React.Component {
   state = {
     posts_arr: [],
     loading: false,
+    titleValue: "",
+    descValue: "",
+    sortValue: "",
+  }
+
+  titleInputHandler = (e) => {
+    this.setState({ titleValue: e.target.value })
+  }
+
+  descInputHandler = (e) => {
+    this.setState({ descValue: e.target.value })
+  }
+
+  cricPostBtnHandler = async (e) => {
+    const id = await createNewPost(this.state.titleValue, this.state.descValue)
+    const newPostArr = await getUniquePost(id)
```

Comment:
If i do this there will be few ms difference on post created time.
Since i set this object in db and but again i have to make new Date() call when i set the state locally rather than making a network request, but the biggest downside with my approach is i cannot get the post id, but in your way of doing things i can get the id. 
`{
    title:title,
    desc:desc,
    createdAt:new Date().getTime()
}`

Reply:
No sir think what you will not work will discuss tomorrow during our session.

Reply:
Why can't you pass createdAt to `createNewPost`. Right now you are setting time inside the helper, just move it up a bit.

Reply:
Will discuss this during the session.